op-guide/backup-restore.md

```diff
@@ -0,0 +1,116 @@
+---
+title: backup and restore
+category: advanced
+---
+
+# Backup and Restore
+
+## About
+
+This document describes how to back up and restore the data of TiDB. Currently, this document only covers full backup and restoration.
+
+Here we assume that the TiDB service information is as follows:
+
+|Name|Address|Port|User|Password|
+|:----:|:-------:|:----:|:----:|:------:|
+|TiDB|127.0.0.1|4000|root|*|
+
+In the process of backing up and restoring data, use the following tools:
+
+- `mydumper`: to export data from TiDB
+- `loader`: to import data into TiDB
+
+### Download TiDB Toolset (Linux)
+
+```bash
+# Download the tool package.
+wget http://download.pingcap.org/tidb-enterprise-tools-latest-linux-amd64.tar.gz
+wget http://download.pingcap.org/tidb-enterprise-tools-latest-linux-amd64.sha256
+
+# Check the file integrity. If the result is OK, the file is correct.
+sha256sum -c tidb-enterprise-tools-latest-linux-amd64.sha256
+
+# Extract the package.
+tar -xzf tidb-enterprise-tools-latest-linux-amd64.tar.gz
+cd tidb-enterprise-tools-latest-linux-amd64
+```
+
+## Full Backup and Restoration Using `mydumper`/`loader`
+
+The `mydumper` is a powerful data backup tool. See more about [`mydumper`](https://github.com/maxbube/mydumper). 
+
+To back up data, use `mydumper` to export data from TiDB. To restore data, use `loader` to import data into TiDB.
+
+**Note:** Although TiDB supports backing up and restoring data using the official `mysqldump` tool of MySQL, its performance is worse than `mydumper`/`loader`. It will take up much time to back up and restore a large amount of data. Therefore, the tool `mysqldump` is not recommended.
+
+### Best Practices of Full Backup and Restoration Using `mydumper`/`loader` 
+
+To quickly back up and restore data (especially large amounts of data), refer to the following recommendations:
+
+- Keep the size of the data file exported using `mydumper` as small as possible. It is recommended to keep it within 64M. You can set the parameter `-F` to 64.
+- You can adjust the `loader` parameter `-t` based on the number of TiKV instances and the load. For example, you can set the parameter to `3 *(1 ～ n)` in the case of 3 TiKV scenarios. When the TiKV load is too high, and a large number of `backoffer.maxSleep 15000ms is exceeded` show in `loader` and TiDB log, adjust it to a smaller value appropriately. When the TiKV load is not too high, adjust it to a higher value appropriately.
+
+#### An Example of Restoring Data and Related Configuration 
+
+- The total amount of data exported using `mydumper` is 214G. Single table 8 columns, 2 billion rows of data.
+- Cluster topology.
+  - TiKV * 12
+  - TIDB * 4
+  - PD * 3
+- Set the `mydumper` parameter `-F` to 16, and `loader` parameter `-t` to 64.
+
+Result: It takes about 11 hours to import, with 19.4G/hour.
```

Comment:
Change this section to:
 - The total size of the exported files is 214G. A single table has 8 columns and 2 billion rows.
 - The cluster topology:
    - 12 TiKV instances: 4 nodes, 3 TiKV instances per node
    - 4 TiDB instances
    - 3 PD instances
 - The configuration of each node:
    - CPU: Intel Xeon E5-2670 v3 @ 2.30GHz
    - 48 vCPU [2 x 12 physical cores]
    - Memory: 128G
    - Disk: sda [raid 10, 300G] sdb[RAID 5, 2T]
    - OS: CentOS 7.2
 - the `-F` parameter of `mydumper` is set to 16 and the `-t` parameter of `loader` is set to 64.

**Results**: It takes 11 hours to import all the data, which is 19.4G/hour.